scripts/files/file_tiff.py

```diff
@@ -221,7 +209,14 @@ def check_no_data(self, gdalinfo: GdalInfo) -> None:
             return
         if "noDataValue" in bands[0]:
             current_nodata_val = bands[0]["noDataValue"]
-            if current_nodata_val != 255:
+            # GDALINFO shows the `noDataValue` as `-9999.0`
```

Comment:
The docs of this function dont mention DEM, the docs also dont actually match the logic as we skip this if there are 4 bands.

but if we have 4 bands and are using DEM then something is really wrong??